src/client/app/components/TooltipHelpComponent.tsx

```diff
@@ -70,7 +70,8 @@ export default function TooltipHelpComponent(props: TooltipHelpProps) {
 		'help.home.toggle.chart.link': { link: `${helpUrl}/chartLink/` },
 		'help.groups.groupdetails': { link: `${helpUrl}/groupViewing/#groupDetails` },
 		'help.groups.groupview': { link: `${helpUrl}/groupViewing/` },
-		'help.meters.meterview': { link: `${helpUrl}/meterViewing/` }
+		'help.meters.meterview': { link: `${helpUrl}/meterViewing/` },
+		'help.admin.unitconversionvisuals': { link: ''}
```

Comment:
Lets set the link to { link: '${helpUrl}/adminUnitVisual/'}. I know it is not there yet but it will get added as part of v2.0.0.